rootfs/etc/profile.d/_preferences.sh

```diff
@@ -13,6 +13,13 @@ fi
 
 [[ -n $_GEODESIC_TRACE_CUSTOMIZATION ]] && echo trace: GEODESIC_CONFIG_HOME is found to be "${GEODESIC_CONFIG_HOME:-<unset>}"
 
+# If HOST_HOME is set, create a symbolic link so host pathnames (at least the ones under $HOME) work inside the shell
```

Comment:
How is `HOST_HOME` different from `LOCAL_HOME` and can we possible consolidate on `LOCAL_HOME`?

Reply:
I didn't look carefully at `LOCAL_HOME` before, but it is what I need and I will use it.